test/unit-tests/mobile/project-files-manager.ts

```diff
@@ -104,10 +104,18 @@ function createTestInjector(): IInjector {
 }
 
 async function createFiles(testInjector: IInjector, filesToCreate: string[]): Promise<string> {
-	let fs = testInjector.resolve("fs");
 	let directoryPath = temp.mkdirSync("Project Files Manager Tests");
 
-	_.each(filesToCreate, file => fs.writeFile(path.join(directoryPath, file), ""));
+	_.each(filesToCreate, file => createFile(testInjector, file, "", directoryPath));
+
+	return directoryPath;
+}
+
+async function createFile(testInjector: IInjector, fileToCreate: string, fileContent: string, directoryPath?: string): Promise<string> {
```

Comment:
this method does not have to be async - it doesn't have any async operations in it